src/main/java/org/assertj/core/api/AbstractZonedDateTimeAssert.java

```diff
@@ -487,6 +496,7 @@ public SELF isNotEqualTo(ZonedDateTime expected) {
    *           String.
    */
   public SELF isNotEqualTo(String dateTimeAsString) {
+    Objects.instance().assertNotNull(info, actual);
```

Comment:
I might miss something but I believe this is not needed as `super.isNotEqualTo` will fail is actual is null since expected is not.

Reply:
Correct, it is not needed. I will add one more unit test for that case.